analyses/fusion_filtering/00-normal-matrix-generation.R

```diff
@@ -0,0 +1,36 @@
+# R. Jin 2021
+# Filter expression matrix containing all specimens to only normal specimens of specific type
+
+suppressPackageStartupMessages(library("optparse"))
+suppressPackageStartupMessages(library("tidyverse"))
+
+
+option_list <- list(
+  make_option(c("-e","--expressionMatrix"),type="character",
+              help="expression matrix (TPM for samples that need to be zscore normalized .RDS)"),
+  make_option(c("-c","--clinicalFile"),type="character",
+              help="Histology file for all samples (.TSV) "),
+  make_option(c("-s","--specimenType"),type="character",
+              help="which specimen type to collect normal expression"),
+  make_option(c("-o","--outputfile"),type="character",
```

Comment:
```suggestion
  make_option(c("-o","--outputFile"),type="character",
```
just to be consistent with other filename options